resync/api/agent_evolution_api.py

```
@@ -548,26 +554,26 @@ async def get_performance_metrics(
     """Load suggestion from disk."""
     import json
-    from pathlib import Path
-
-    file_path = Path("data/agent_improvements") / f"{suggestion_id}.json"
+    file_path = AGENT_IMPROVEMENTS_DIR / f"{suggestion_id}.json"
 
     if not file_path.exists():
         return None
 
     try:
+        if not AGENT_IMPROVEMENTS_DIR.exists():
```

Ruff (0.15.0)

[warning] Async functions should not use pathlib.Path methods, use trio.Path or anyio.path

(ASYNC240)